t/03-transactions.t

```diff
@@ -34,6 +34,7 @@ __DATA__
             local txn = require("resty.lmdb.transaction")
             local l = require("resty.lmdb")
 
+            l.set("testbalabala", "aaaa")
```

Comment:
Does this line mean something? or just a debug line?

Reply:
It adds some field to lmdb, I think it makes sense.

Reply:
i'm also curious on this change, is there anything start to break without this change?